projects/igniteui-angular/src/lib/time-picker/time-picker.component.html

```diff
@@ -1,95 +1,72 @@
+        [displayFormat]="this.displayFormat" [igxTextSelection]="this.isDropdown" role="combobox" aria-haspopup="dialog"
+        [attr.aria-expanded]="!this.toggleDirective.collapsed" [attr.aria-labelledby]="this.label?.id" />
 
-<ng-template #dropdownInputTemplate>
-    <igx-input-group #group (mousedown)="mouseDown($event)" [suppressInputAutofocus]="true">
-        <label igxLabel *ngIf="!labelDirective">Time</label>
-        <ng-container ngProjectAs="[igxLabel]" *ngTemplateOutlet="labelTemplate"></ng-container>
-        <igx-prefix (click)="openDialog(group.element.nativeElement)">
-            <igx-icon>access_time</igx-icon>
-        </igx-prefix>
-        <input
-            type="text"
-            [igxMask]="mask"
-            igxInput
-            [includeLiterals]="true"
-            [placeholder]="format"
-            [displayValuePipe]="displayFormat"
-            [focusedValuePipe]="inputFormat"
-            [promptChar]="promptChar"
-            [value]="displayValue"
-            [igxTextSelection]="true"
-            (input)="onInput($event)"
-            (blur)="onBlur($event)"
-            (focus)="onFocus($event)"
-            (wheel)="spinOnEdit($event)"
-            [disabled]="disabled" />
-        <igx-suffix *ngIf="showClearButton" igxRipple (click)="clear()">
-            <igx-icon>clear</igx-icon>
-        </igx-suffix>
-    </igx-input-group>
-</ng-template>
-<ng-template #defaultTimePickerTemplate>
-    <igx-input-group (click)="openDialog()" (mousedown)="mouseDown($event)">
-        <igx-prefix>
-            <igx-icon>access_time</igx-icon>
-        </igx-prefix>
-        <label igxLabel *ngIf="!labelDirective">Time</label>
-        <ng-container ngProjectAs="[igxLabel]" *ngTemplateOutlet="labelTemplate"></ng-container>
-        <input
-            igxInput
-            [value]="displayTime || ''"
-            [disabled]="disabled"
-            tabindex="0" readonly
-            (blur)="onBlur($event)" />
-    </igx-input-group>
-</ng-template>
-<ng-container *ngTemplateOutlet="template; context: context"></ng-container>
+    <ng-container ngProjectAs="[igxLabel]">
+        <ng-content select="[igxLabel]"></ng-content>
+    </ng-container>
+    <ng-container ngProjectAs="igx-prefix">
+        <ng-content select="igx-prefix,[igxPrefix]"></ng-content>
+    </ng-container>
+    <ng-container ngProjectAs="igx-suffix">
+        <ng-content select="igx-suffix,[igxSuffix]"></ng-content>
+    </ng-container>
+    <ng-container ngProjectAs="igx-hint">
+        <ng-content select="igx-hint,[igxHint]"></ng-content>
+    </ng-container>
+
+    <igx-prefix *ngIf="!this.toggleComponents.length" (click)="this.toggle()">
+        <igx-icon fontSet="material">access_time</igx-icon>
+    </igx-prefix>
+
+    <igx-suffix *ngIf="this.showClearButton" igxRipple (click)="this.clear()">
+        <igx-icon fontSet="material">clear</igx-icon>
+    </igx-suffix>
```

Comment:
Those are likely not supposed to be together, this will put the default open icon _after_ user projected prefixes. The initial comment was a question about the suffix, though I'm not 100% on that either. 
Anyway, if there's no design where the default icons sit in relation to user-provided ones, do consult w/ Stefan if needed and place the defaults as the design dictates :)